requirements/extra.txt

```diff
@@ -7,4 +7,4 @@ torchtext>=0.3.1, <0.7  # TODO: temporary fix fix for compatibility
 onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
-https://github.com/facebookresearch/fairscale/archive/8e85ce8c93569017521d92ceb78dba2c57c955a0.zip # TODO temporary fix till release version
+https://github.com/tchaton/fairscale/archive/dev.zip
```

Comment:
I think this is not supported by pip. AFAIk, when releasing this to pip, you can only include dependencies here, that are on pip themselves

Reply:
Not sure to follow there, but it does work 👍